mail-connector/src/main/java/eu/tsystems/mms/tic/testframework/mailconnector/util/EmailQuery.java

```diff
@@ -42,7 +42,7 @@ public class EmailQuery implements PropertyManagerProvider {
     private static final String MAX_READ_TRIES_PROPERTY = "MAX_READ_TRIES";
 
     private int retryCount = Integer.parseInt(PROPERTY_MANAGER.getProperty(MAX_READ_TRIES_PROPERTY, "20"));
-    private long pauseMs = Integer.parseInt(PROPERTY_MANAGER.getProperty(POLLING_TIMER_SECONDS_PROPERTY, "10")) * 1000L;
+    private long pauseMs = PROPERTY_MANAGER.getLongProperty(POLLING_TIMER_SECONDS_PROPERTY, "10") * 1000;
```

Comment:
Default value should not `"10"`, but `10_000L` and the calculation with `* 1000` ist not needed any more.